Documentation/ceph-cluster-crd.md

```diff
@@ -444,6 +452,61 @@ You can set priority class names for Rook components for the list of key value p
 
 The specific component keys will act as overrides to `all`.
 
+### Health settings
+
+Rook-Ceph will monitor the state of the CephCluster on various components by default.
+The following CRD settings are available:
+
+* `healthCheck`: main ceph cluster health monitoring section
+
+Currently three health checks are implemented:
+
+* `mon`: health check on the ceph monitors, basically check whether monitors are members of the quorum. If after a certain timeout a given monitor has not joined the quorum back it will be failed over and replace by a new monitor.
+* `osd`: health check on the ceph osds
+* `status`: ceph health status check, periodically check the Ceph health state and reflects it in the CephCluster CR status field.
+
+The liveness probe of each daemon can also be controlled via `livenessProbe`, the setting is valid for `mon`, `mgr` and `osd`.
+Here is a complete example for both `daemonHealth` and `livenessProbe`:
+
+```yaml
+healthCheck:
+  daemonHealth:
+    mon:
+      disabled: false
+      interval: 60s
```

Comment:
Should these be the mon settings?
```yaml
    mon:
      disabled: false
      interval: 45s
      timeout: 600s
```

Reply:
Yes.

Reply:
How about keeping the docs the same as the default in the cluster.yaml?
```suggestion
      interval: 45s
```